DatadogSDKCrashReporting.podspec.src

```diff
@@ -1,27 +0,0 @@
-Pod::Spec.new do |s|
-  s.name         = "DatadogSDKCrashReporting"
-  s.module_name  = "DatadogCrashReporting"
-  s.version      = "__DATADOG_VERSION__"
-  s.summary      = "Official Datadog Crash Reporting SDK for iOS."
-  
-  s.homepage     = "https://www.datadoghq.com"
-  s.social_media_url   = "https://twitter.com/datadoghq"
-
-  s.license            = { :type => "Apache", :file => 'LICENSE' }
-  s.authors            = { 
-    "Maciek Grzybowski" => "maciek.grzybowski@datadoghq.com",
-    "Mert Buran" => "mert.buran@datadoghq.com",
-    "Maxime Epain" => "maxime.epain@datadoghq.com"
-  }
-
-  s.swift_version      = '5.1'
-  s.ios.deployment_target = '11.0'
-  s.tvos.deployment_target = '11.0'
-
-  s.source = { :git => 'https://github.com/DataDog/dd-sdk-ios.git', :tag => s.version.to_s }
-  s.static_framework = true
-
-  s.source_files = "Sources/DatadogCrashReporting/**/*.swift"
-  s.dependency 'DatadogSDK', '__DATADOG_VERSION__'
```

Comment:
We need to change this line in the original `.podspec` in order to make it work. The solution we're replacing was depending on `__DATADOG_VERSION__` token replacement, while the new one modifies `s.version      = "__DATADOG_VERSION__"`.

If we change it to:
```
  s.dependency 'DatadogSDK', s.version.to_s
```
in the original `podspec`, it should be fine.

Reply:
oh damn, that right!